### DISCUSSION TITLE
Help me save this issue with Github clone

### DISCUSSION THREAD

**Discussion by tigertigerpi** [vercel/community#3344] [Bug Report]
### Page to Investigate

https://vercel.com/new/clone?repository-url=https://github.com/oleeskild/digitalgarden

### Steps to Reproduce

When I was deploying git repository, there is this error:

An unexpected error occurred. Our team has already been notified and are working to resolve the issue, please try again shortly.

<img width="1463" alt="image" src="https://github.com/vercel/community/assets/138350800/0d0103ef-2ba7-4c48-bafb-0d9e04a1acb6">

This is very important to me , Please help me solve this problem. Thank you!

**Comment by 0xw01f** [comment]
Hi, have you tried to look atthe javascript console in your browser ?
I had the same error, this is what saved me.

**Comment by amyegan** [comment]
I was not able to replicate this problem. Is it still happening for you?
